mne/minimum_norm/tests/test_time_frequency.py

```diff
@@ -129,9 +130,152 @@ def test_tfr_with_inverse_operator(method):
+    assert vlen_lh + 1 == vlen_rh == 3
+
+    # prepare instances of BiHemiLabel
+    fname_lvis = data_path / "MEG" / "sample" / "labels" / "Vis-lh.label"
+    fname_rvis = data_path / "MEG" / "sample" / "labels" / "Vis-rh.label"
+    lvis = read_label(fname_lvis)
+    rvis = read_label(fname_rvis)
+    bihl = BiHemiLabel(lh=label, rh=label2)  # auditory labels
+    bihl.name = "Aud"
+    bihl2 = BiHemiLabel(lh=lvis, rh=rvis)  # visual labels
+    bihl2.name = "Vis"
+    bihls = [bihl, bihl2]
+    bad_bihl = BiHemiLabel(lh=bad_lab, rh=rvis)  # 1 unique vert on lh, rh ok
+    bad_bihls = [bihl, bad_bihl]
+    print("BiHemi label verts:", bihl.lh.vertices.shape, bihl.rh.vertices.shape)
+
+    label_sets = [[labels, bad_lbls], [bihls, bad_bihls]]
+
+    # check error handling
+    sip_kwargs = dict(
+        # baseline=(-0.1, 0),
+        # baseline_mode="percent",
+        baseline=(-0.1, 0),
+        baseline_mode="mean",
+        n_cycles=2,
+        n_jobs=None,
+        return_plv=False,
+        method="dSPM",
+        prepared=True,
+    )
+    # label input errors
+    with pytest.raises(TypeError, match="must be an instance of"):
+        source_induced_power(epochs, inv, freqs, label="bad_input", **sip_kwargs)
+    with pytest.raises(TypeError, match="must be an instance of"):
+        source_induced_power(
+            epochs, inv, freqs, label=[label, "bad_input"], **sip_kwargs
+        )
+
+    # error handling for multi-label and plv
+    with pytest.raises(RuntimeError, match="value cannot be calculated"):
+        source_induced_power(
+            epochs,
+            inv,
+            freqs,
+            labels,
+            baseline=(-0.1, 0),
+            baseline_mode="percent",
+            n_cycles=2,
+            n_jobs=None,
+            return_plv=True,
+            method="dSPM",
+            prepared=True,
+        )
+
+    # check multi-label handling
+    for ltype, lab_set in zip(("Label", "BiHemi"), label_sets):
+        vlen = vlen_lh if ltype == "Label" else vlen_lh + vlen_rh
+        # check overlapping verts error handling
+        with pytest.raises(RuntimeError, match="overlapping vertices"):
+            source_induced_power(epochs, inv, freqs, lab_set[1], **sip_kwargs)
+
+        for ori in (None, "normal"):  # check loose and normal orientations
+            sip_kwargs.update(pick_ori=ori)
+            lbl = lab_set[0][0]
+
+            # check label=Label vs label=[Label]
+            no_list_pow = source_induced_power(
+                epochs, inv, freqs, label=lbl, **sip_kwargs
+            )
+            assert no_list_pow.shape == (vlen, flen, tlen)
+
+            list_pow = source_induced_power(
+                epochs, inv, freqs, label=[lbl], **sip_kwargs
+            )
+            assert list_pow.shape == (1, flen, tlen)
+
+            nlp_ave = np.mean(no_list_pow, axis=0)
+            assert_allclose(nlp_ave, list_pow[0], rtol=1e-3)
+
+            # check label=[Label, Label]
```

Comment:
```suggestion
            # check label=[Label1, Label2]
```
Initially I got the wrong impression that this was testing what happens when you pass the same label twice